maat-court-data-api/src/test/java/gov/uk/courtdata/applicant/service/ApplicantHistoryServiceTest.java

```diff
@@ -30,20 +31,36 @@ public class ApplicantHistoryServiceTest {
     @InjectMocks
     private ApplicantHistoryService applicantHistoryService;
 
+    @Test
+    void givenAValidInput_whenFindIsInvoked_thenShouldReturnApplicantHistoryDTO() {
+        when(applicantHistoryRepository.findById(anyInt())).thenReturn(Optional.of(ApplicantHistoryEntity.builder().id(ID).build()));
+        applicantHistoryService.find(ID);
+        verify(applicantHistoryRepository, atLeastOnce()).findById(ID);
```

Comment:
This is superfluous, if we're mocking with `when`, it makes the `verify` call redundant. I think we should either mock the call to the mapper too and check the DTO that gets returned, or eliminate the `when` entirely.